README.md

```diff
@@ -136,6 +136,8 @@ python3 setup.py install
 python3 setup.py develop
 ```
 
+If the commands above don't work maybe you need to install the `python-devel` and `gcc-c++` packages.
```

Comment:
```suggestion
If the commands above don't work maybe you need a C++ compiler and python headers.
```

@rporres , Does that covers your suggestions?

Reply:
indeed, thx!